### PR TITLE
Update threat-protection-integration.md

### PR DESCRIPTION
**Pull request by msbemba**
Fixed broken link

### CODE REVIEW

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/5224

**Comment by msbemba** [comment]
@JohanFreelancer9  Please copy review

File: microsoft-365/security/defender-endpoint/threat-protection-integration.md [hunk 1]
```diff
@@ -63,7 +63,7 @@ The Skype for Business integration provides a way for analysts to communicate wi
 ## Microsoft 365 Defender
 With Microsoft 365 Defender, Microsoft Defender for Endpoint and various Microsoft security solutions form a unified pre- and post-breach enterprise defense suite that natively integrates across endpoint, identity, email, and applications to detect, prevent, investigate and automatically respond to sophisticated attacks.
```
Comment (JohanFreelancer9) [inline microsoft-365/security/defender-endpoint/threat-protection-integration.md]:
```suggestion
With Microsoft 365 Defender, Microsoft Defender for Endpoint, and various Microsoft security solutions form a unified pre- and post-breach enterprise defense suite that natively integrates across endpoint, identity, email, and applications to detect, prevent, investigate, and automatically respond to sophisticated attacks. 
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @msbemba, a couple of minor additional edits, please.

**Pull request merged**